<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sudoku</class>
 <widget class="QMainWindow" name="Sudoku">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/sudoku1.png</normaloff>:/img/sudoku1.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gLTablero">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0" rowspan="10" colspan="3">
       <layout class="QGridLayout" name="glPistas"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pBf3">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/3.png</normaloff>:/img/3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pBf4">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/4.png</normaloff>:/img/4.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLCDNumber" name="contTiempo"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pBf1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/1.png</normaloff>:/img/1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pBf2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/2.png</normaloff>:/img/2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pBf5">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/5.png</normaloff>:/img/5.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pBf6">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/6.png</normaloff>:/img/6.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pBf7">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/7.png</normaloff>:/img/7.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pBf8">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/8.png</normaloff>:/img/8.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pBf9">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/9.png</normaloff>:/img/9.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btHelp">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>true</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white</string>
        </property>
        <property name="text">
         <string>FICHAS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>true</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white</string>
        </property>
        <property name="text">
         <string>AYUDA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>true</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white</string>
        </property>
        <property name="text">
         <string>TIEMPO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>true</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white</string>
        </property>
        <property name="text">
         <string>PISTAS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_Juego"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionNuevo_Juego">
   <property name="text">
    <string>Nuevo Juego</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionMostrar_Solucion">
   <property name="text">
    <string>Mostrar Solucion</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar Partida</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
